code/game/machinery/doors/airlock.dm

```diff
@@ -324,7 +324,9 @@ About the new airlock wires panel:
 /obj/machinery/door/airlock/examine(mob/user)
 	..()
 	if(charge && !p_open && in_range(user, src))
-		user << "The maintenance panel is bulging slightly."
+		user << "<span class='warning'>The maintenance panel is slightly bulging.</span>"

```

Comment:
the fluff behind it is that the charge is bulky and they don't -completely- fasten the panel, just kinda slap it on